### PR TITLE
Removing frontend Ember app (i.e. datahub-web folder)

### PR DESCRIPTION
**Pull request by kafkahw**
Removing frontend ember app (i.e. `datahub-web` folder) as announced in DataHub [slack channel](https://datahubspace.slack.com/archives/CUMV92XRQ/p1624642329048700) one month ago. 

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by kafkahw** [comment]
@shirshanka @jjoyce0510 @gabe-lyons I think I've fixed the doc generation issue. Running `./gradlew :docs-website:yarnBuild` locally seems to be fine now. Can you guys please help approve running workflows? I think as a first-time contributor I can't approve it.

File: datahub-frontend/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # DataHub Frontend
 DataHub frontend is a [Play](https://www.playframework.com/) service written in Java. It is served as a mid-tier
-between [DataHub GMS](../gms) which is the backend service and [DataHub Web](../datahub-web).
```
Comment (hsheth2) [inline datahub-frontend/README.md:3]: Would be better to change the link to reference datahub-web-react/README.md

**Review by hsheth2 — APPROVED** [review]
Assuming the build passes, LGTM

Super exciting!!

File: datahub-frontend/README.md (hunk 1 above)
Comment (kafkahw) [reply datahub-frontend/README.md:3]: Good suggestion! Updated in [the latest commit](https://github.com/linkedin/datahub/pull/2921/commits/d54f0134d3e93a21a61ecf9cd3727175192c1c55)

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**